watermarker/src/commonTest/kotlin/unitTest/watermarks/TextWatermarkTest.kt

```diff
@@ -68,6 +68,50 @@ class TextWatermarkTest {
         assertEquals(textBytes, content.value)
     }
 
+    @Test
+    fun small_loremIpsum_compression() {
+        val customText = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr"
+        val customTextBytes = customText.encodeToByteArray().asList()
+
+        // Arrange
+        val expectedTrendmark = CompressedRawTrendmark.new(customTextBytes)
+
+        // Act
+        val textWatermark = TextWatermark.small(customText)
+        val trendmark = textWatermark.finish()
+        val content = trendmark.getContent()
+
+        // Assert
+        assertEquals(customText, textWatermark.text)
+        assertFalse(textWatermark.isSized())
+        assertTrue(textWatermark.isCompressed())
+        assertEquals(expectedTrendmark, trendmark)
+        assertTrue(content.isSuccess)
+        assertEquals(customTextBytes, content.value)
+    }
+
+    @Test
+    fun small_loremIpsum_noCompression() {
+        val customText = "Lorem"
+        val customTextBytes = customText.encodeToByteArray().asList()
+
+        // Arrange
+        val expectedTrendmark = RawTrendmark.new(customTextBytes)
```

Comment:
Everything in this block is part of the `// Arrange`. So start with the `// Arrange` comment.

Further, there is a `text` variable already defined in line 30 with its Byte representation (`textBytes`). Check if this can be used either here or in the other test.

Reply:
I relocated the `// Arrange` comment now.

I did not use the given `text` and `textBytes` variables as I need one text that is certainly compressible (`small_loremIpsum_compression`) and one that is certainly not compressible (`small_loremIpsum_noCompression`). I could use use the given text for one scenario if you want me to (I'd have to check which one as I don't know if it is compressible or not). For consistency and clarity reasons I did not do so yet.